.github/workflows/build-azure-cli-base.yaml

```
@@ -0,0 +1,47 @@
+        uses: docker/setup-buildx-action@v3
+
+      - name: Log in to registry.ci.openshift.org
+        uses: docker/login-action@v3
+        with:
+          registry: registry.ci.openshift.org
+          username: ${{ secrets.REGISTRY_CI_USERNAME }}
+          password: ${{ secrets.REGISTRY_CI_PASSWORD }}
+
+      - name: Generate immutable image tag
+        id: image-tag
+        run: echo "tag=v$(date +%Y%m%d)" >> $GITHUB_OUTPUT
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:32: Double quote to prevent globbing and word splitting

(shellcheck)